filebeat/docs/inputs/input-journald.asciidoc

```diff
@@ -137,10 +137,31 @@ If you have old log files and want to skip lines, start {beatname_uc} with
 [id="{beatname_lc}-input-{type}-include-matches"]
 ==== `include_matches`
 
-A list of filter expressions used to match fields. The format of the expression
+A collection of filter expressions used to match fields. The format of the expression
 is `field=value`. {beatname_uc} fetches all events that exactly match the
 expressions. Pattern matching is not supported.
 
+If you configured a filter expression, only entries with this field set will be iterated.
```

Comment:
I kept it like this on purpose because I wanted to provide information about how the reader of Filebeat interacts with the journals. Probably, I could have done a better job in this section and added more context.